openwisp_radius/tests/test_admin.py

```diff
@@ -1129,8 +1129,57 @@ def test_get_is_verified_user_admin_list(self):
         response = self.client.get(url)
 
         def get_expected_html(value):
-            return f'<td class="field-get_is_verified"><img src="/static/admin/img/icon-{value}.svg" alt="{value}">'
+            return (
+                '<td class="field-get_is_verified"><img src="/static/admin/img/icon-'
+                f'{value}.svg" alt="{value}">'
+            )
 
         self.assertContains(response, get_expected_html('yes'))
         self.assertContains(response, get_expected_html('no'))
         self.assertContains(response, get_expected_html('unknown'))
+
+    def test_registered_user_filter(self):
+        unknown = User.objects.first()
+        self.assertIsNotNone(unknown)
+        verified = User(
+            username='verified', password='verified', email='verified@test.com'
+        )
+        verified.full_clean()
+        verified.save()
+        RegisteredUser.objects.create(
+            user=verified, identity_verification='mobile_phone', is_verified=True
+        )
+        unverified = User.objects.create(
+            username='unverified', password='unverified', email='unverified@test.com'
+        )
+        unverified.full_clean()
+        unverified.save()
+        RegisteredUser.objects.create(
+            user=unverified, identity_verification='mobile_phone', is_verified=False
+        )
+        app_label = User._meta.app_label
+        url = reverse(f'admin:{app_label}_user_changelist')
+
+        def get_expected_html(value):
+            return (
+                '<td class="field-get_is_verified"><img src="/static/admin/img/icon-'
+                f'{value}.svg" alt="{value}">'
+            )
+
+        with self.subTest('Verified users filter'):
+            response = self.client.get(f'{url}?is_verified=true')
```

Comment:
Try passing the querystring parameters as a dict, it should work:
```python
self.client.get(url, {'is_verified': 'true'})
```

It's a lot cleaner. It's better to use this form (see also https://github.com/openwisp/openwisp-users/issues/242).